server/server_test.go

```diff
@@ -1211,6 +1211,7 @@ func TestConfigV1(t *testing.T) {
 	expected["services"] = map[string]interface{}{"acmecorp": map[string]interface{}{"url": "https://example.com/control-plane-api/v1"}}
 	expected["default_authorization_decision"] = "/system/authz/allow"
 	expected["default_decision"] = "/system/main"
+	expected = map[string]interface{}{"result": expected}
```

Comment:
🙈

Reply:
Fixed. There must be a German word for this.